src/Command/Ide/IdeOpenCommand.php

```diff
@@ -35,9 +35,6 @@ protected function configure() {
   protected function execute(InputInterface $input, OutputInterface $output) {
     $acquia_cloud_client = $this->cloudApiClientService->getClient();
     $cloud_application_uuid = $this->determineCloudApplication();
-    if (!$cloud_application_uuid && !$input->isInteractive()) {
-      throw new RuntimeException('Not enough arguments (missing: "applicationUuid").');
```

Comment:
This is the only place in the codebase where the command validates that an application was found. I think it's safe to assume that if any command calls `determineCloudApplication()`, an application _must_ be found or an exception thrown.